configuration/enable-offline-mode.adoc

```diff
@@ -46,14 +46,17 @@ include::modules/enable-offline-mode-roxctl.adoc[leveloffset=+2]
 [id="update-scanner-definitions-in-offline-mode"]
 == Updating Scanner definitions in offline mode
 
-Scanner contains a local vulnerability definitions database. When {product-title} runs in normal mode (connected to the internet), Scanner fetches new vulnerability definitions from the internet and updates its database.
+Scanner contains a local vulnerability definitions database. If {rh-rhacs-first} is running in normal mode, Scanner fetches new vulnerability definitions from Central. Central retrieves the updates from the internet, and then updates the Central database.
 //TODO: Add link to Vulnerability definitions
 
-However, when you are using {product-title} in offline mode, you must manually update Scanner definitions by uploading them to Central.
+However, if you are using {product-title-short} in offline mode, you must manually update the definitions. 
+By uploading the definitions to Central, you can manually update the definitions.
```

Comment:
I really don't like that we have been using "definitions" in our product, and I want to remove it.

```suggestion
However, if you are using {product-title-short} in offline mode, you must manually update the vulnerability data by uploading a bundle to Central, which Scanner will fetch.
```